### PR TITLE
PoC to download the sources

### PR DESCRIPTION
**Pull request by ignasi35**
References #61

To download sources we have some options:

1. (this PR) manually zip, add to git and use antora/asciidoc relative links to [`attachments`](https://docs.antora.org/antora/2.3/asciidoc/link-attachment/)
2. automatically (GitHub action?) build zips and attach the binaries to a GitHub release. HTML uses absolute URLs with the necessary placeholder (tag version, etc...)

PS: To be solved in a separate PR [`filter out the tags`](https://github.com/akka/akka-platform-guide/issues/61#issuecomment-679015361)

### CODE REVIEW

**Comment by patriknw** [comment]
Another option is to use TechHub in same way as for akka-samples, ie it creates a zip from a directory. That could also be on a branch if we don’t want to publish each commit and would like to do some scriped pre-prosessing such as removing the doc tags.

File: docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc [hunk 1]
```diff
@@ -118,6 +118,10 @@ You can stop the service with `ctrl-c`.
 
 ifdef::todo[TODO: deploy to cloud and try it]
 
+== Get the source code
+
+link:{attachmentsdir}/01-shopping-cart-service-scala.zip[Download complete project for this step] and run it.
```
Comment (patriknw) [inline docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:123]: I was originally thinking that we would have one single zip with directories for each step. Pros, cons?

Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:123]: Having a single ZIP will simplify things on our end but may confuse users. Using more specific downloads will reduce the chances of mistakes (¿increasing engaging?). Can't think of many other pro/cons.

I thought of the use case where a user only interested in `step 3: persistent entity` could start off using a download that contains _only_ what they need.

So, there would potentially be links to download code at:

1. the beginning of the `step 3: persistent entity` (only has gRPC code)
2. the beginning of each of the projection steps (has gRPC and the complete entity)
3. the end of each of the projection steps (the complete shopping cart and, if applicable, one of the other services)

Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:123]: Maybe the simplest thing we can do is to expand the doc deploy script to zip (and cleanup) the sources, and upload them together with the documentation.

Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:123]: > Maybe the simplest thing we can do is to expand the doc deploy script to zip (and cleanup) the sources, and upload them together with the documentation.

SGTM. Makes the site less dependant on external tools (e.g. techhub)

Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/grpc-server.adoc:123]: Done in #240

**Comment by ignasi35** [comment]
Closed in favor of

**Pull request closed without merging**